lib/Header/Header.js

```diff
@@ -616,6 +616,7 @@ module.exports = kind(
 	* @private
 	*/
 	typeChanged: function () {
+		this.addRemoveClass('moon-large-header', this.get('type') == 'large');
```

Comment:
![Yusssss](http://images6.fanpop.com/image/photos/33800000/yus-adventure-time-with-finn-and-jake-33881905-500-281.gif)